homeassistant/components/air_quality/group.py

```diff
@@ -1,12 +1,16 @@
 """Describe group states."""
 
-from homeassistant.components.group import GroupIntegrationRegistry
+from typing import TYPE_CHECKING
+
 from homeassistant.core import HomeAssistant, callback
 
+if TYPE_CHECKING:
+    from homeassistant.components.group import GroupIntegrationRegistry
```

Comment:
Guarded these with if `TYPE_CHECKING` as I'm trying to clean up all the circular imports in group https://github.com/home-assistant/core/pull/113554